UIKitDZ/UIKitDZ/PlayerViewController.swift

```diff
@@ -0,0 +1,131 @@
+// PlayerViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+//
+//  PlayerViewController.swift
+//  UIKitDZ
+//
+//  Created by Vermut xxx on 12.02.2024.
+//
+import AVFoundation
+import UIKit
+
+/// Вью контроллер плеера второго экрана приложения
+final class PlayerViewController: UIViewController {
+    // MARK: - Private Properties
+
+    private var player: AVAudioPlayer?
+    private var currentTrackIndex = 0
+    private var timer: Timer?
+
+    // MARK: - Properties
+
+    var track: String?
+    var tracks: [String]?
+
+    // MARK: - IBOutlets
+
+    @IBOutlet var trackTimeLabel: UILabel!
+    @IBOutlet var trackNameLabel: UILabel!
+    @IBOutlet var slider: UISlider!
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
```

Comment:
Разгрузи его, просто в другую функцию все, что там есть

Reply:
```suggestion
       override func viewDidLoad() {
        super.viewDidLoad()
        setupVolumeButtons()
        playMusic()
        setupBackground()

        // слайдер
        slider.minimumValue = 0
        if let duration = player?.duration {
            slider.maximumValue = Float(duration)
        }
        slider.addTarget(self, action: #selector(sliderValueChanged(_:)), for: .valueChanged)
        startUpdatingSlider()
    }

```
если выношу настройку слайдера в отдельную функцию, то у меня он перестает обновляться почему то